packages/contracts/src/dollar/core/StakingShare.sol

```diff
@@ -113,6 +117,20 @@ contract StakingShare is ERC1155Ubiquity {
         _totalLP += lpDeposited;
     }
 
+    /**
+     * @dev See {IERC1155-safeTransferFrom}.
+     */
+    function safeTransferFrom(
```

Comment:
We should remove this method because it is already implemented here https://github.com/ubiquity/ubiquity-dollar/blob/4b1e1516df5bc41b68c0fcf9083743122fb32933/packages/contracts/src/dollar/core/ERC1155Ubiquity.sol#L135